content/docs/atom-guide/look-dev/materials/material-type-file-spec.md

```diff
@@ -6,65 +6,488 @@ toc: true
+                        }
+                    }
+                ],
+                "functors": [
+                    {
+                        "type": "UseTexture",
+                        "args": {
+                            "textureProperty": "textureMap",
+                            "useTextureProperty": "useTexture",
+                            "dependentProperties": ["textureMapUv"],
+                            "shaderOption": "o_specularF0_useTexture"
+                        }
+                    }
+                ]
+            },
+            { 
+                "$import": "MaterialInputs/SubsurfaceAndTransmissionPropertyGroup.json"
+            },
+            { 
+                "name": "general",
+                "displayName": "General Settings",
+                "description": "General settings.",
+                "properties": [
+                    {
+                        "name": "applySpecularAA",
+                        "displayName": "Apply Specular AA",
+                        "description": "Whether to apply specular anti-aliasing in the shader.",
+                        "type": "Bool",
+                        "defaultValue": false,
+                        "connection": {
+                            "type": "ShaderOption",
+                            "name": "o_applySpecularAA"
+                        }
+                    },
+                    {
+                        "name": "enableMultiScatterCompensation",
+                        "displayName": "Multiscattering Compensation",
+                        "description": "Whether to enable multiple scattering compensation.",
+                        "type": "Bool",
+                        "connection": {
+                            "type": "ShaderOption",
+                            "name": "o_enableMultiScatterCompensation"
+                        }
+                    }
+                ]
+            }
+        ]
+    },
+...
+```
+
+## **version** (deprecated)
+This version number in the `propertyLayout` section is no longer used and has been replaced by the [`version`](#version) number above.
+
+## **groups** (deprecated)
+This is replaced by `propertyGroups` (in [propertyLayout](#propertylayout)), which can support any number of group levels with properties and groups defined in the same place. The engine can still load the old format but any new material type files should use the new format.
 
 {{< note >}}
-  The `general` group is built-in and always available, even if there is no `general` group defined in the `groups` section. This group is typically used to contain any basic properties of this material type.
+An older version of the material type file format organized propertyLayout with separate `groups` and `properties` sections. This imposed several limitations like restricting property grouping to only one level and making it difficult to factor out reusable property groups.
```

Comment:
Removed the comma because "restricting property grouping to only one level" describes the whole limitation. Otherwise, with the comma, it seems to indicate that "to only one level" applies to "This imposed several limitations". 

```suggestion
An older version of the material type file format organized `propertyLayout` with separate `groups` and `properties` sections. This imposed several limitations, such as restricting property grouping to only one level. This made it difficult to factor out reusable property groups.
```